packages/nhost-js/src/clients/functions/index.ts

```diff
@@ -80,7 +80,7 @@ export class NhostFunctionsClient {
 
       let data: T
 
-      if (result.headers.get('content-type') === 'application/json') {
+      if (result.headers.get('content-type')?.includes('application/json')) {
```

Comment:
the server sometimes returns this header as `application/json; <some-extra-info>` so we can't always assume that it strictly equals `application/json`